### PR TITLE
Upgrade tool chain

### PR DESCRIPTION
**Pull request by alexduf**
fixes the errors encountered during release

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -32,20 +32,20 @@ scmInfo := Some(ScmInfo(
...
-      <id>robertberry</id>
-      <name>Robert Berry</name>
-      <url>https://github.com/robertberry</url>
-    </developer>
-  </developers>
-)
+    <developers>
+      <developer>
+        <id>robertberry</id>
+        <name>Robert Berry</name>
```
Comment (agulowaty) [inline build.sbt]: time to change to actual developers ?

File: build.sbt [hunk 2]
```diff
@@ -21,9 +21,9 @@ libraryDependencies ++= Seq(
   "com.github.tomakehurst" % "wiremock" % "1.33" % "test"
 )
 
-releaseSettings
+//releaseSettings
 
-sonatypeSettings
+//sonatypeSettings
```
Comment (agulowaty) [inline build.sbt]: please remove instead commenting out

**Comment by agulowaty** [comment]
:+1:

File: build.sbt (hunk 2 above)
Comment (alexduf) [inline build.sbt]: woups :)

**Pull request merged**